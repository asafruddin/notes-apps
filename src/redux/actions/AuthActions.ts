import { LOGIN } from "../../constant/ActionTypes"

export const StoreKey = (biometricKey: String) => {    
    return ({
        type: LOGIN,
        key: biometricKey,
    })
}