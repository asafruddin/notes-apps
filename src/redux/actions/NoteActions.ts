import { ADD_NOTES, UPDATE_NOTES } from "../../constant/ActionTypes";

export const AddNotes = (title: String, text: String, id: number) => {
    
    return ({
        type: ADD_NOTES,
        title: title,
        text: text,
        id: id
    });
}

export const UpdateNote = (title: String, text: String, id: number) => {
    return ({
        type: UPDATE_NOTES,
        title: title, 
        text: text,
        id: id
    })
}