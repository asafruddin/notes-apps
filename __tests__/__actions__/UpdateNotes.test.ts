import { describe, it } from "@jest/globals";
import { configureStore } from "@reduxjs/toolkit";
import rootReducer from "../../src/redux/reducers/RootReducer";
import { StoreKey } from "../../src/redux/actions/AuthActions";
import { UpdateNote } from "../../src/redux/actions/NoteActions";

describe('testing add biometric and login actions', () => {
    const store = configureStore({
        reducer: rootReducer
    })

    it('user login should be success', async () => {
        await store.dispatch(UpdateNote('title', 'text', 9))
        expect(store.getState).toMatchSnapshot()
    })
})