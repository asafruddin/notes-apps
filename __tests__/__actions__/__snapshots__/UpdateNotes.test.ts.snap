// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing add biometric and login actions user login should be success 1`] = `[Function]`;
